pydoof/helpers.py

```diff
@@ -4,6 +4,18 @@
 from collections import Iterable
 from datetime import date
 from enum import Enum
+from typing import Any, List
+
+
+def _has_dicts(values: List[Any]):
+    """
+    This function checks if a list of values contains dictionaries.
+
+    * Performance improvement:
+
+    Could be possible to check only the first element. Is used on facets and sort, for example.
+    """
+    return any(isinstance(value, dict) for value in values)
```

Comment:
Just a small note here. We have no document regarding functions order, but we usually write the "public" functions at the beginning of the module, and the "private" ones at the bottom. There are no real private functions in python, but this is an auxiliary function that is going to be uses just here, so it should be at the bottom. Also, there is no need to document this kind of functions. There are some simple that the name itself should be enough. You can keep the comment _"Could be possible to check only the first element. Is used on facets and sort, for example."_ but as a code comment and not as a docstring.

Reply:
Sorry I dind't see it. Take a look to the new PR :wink: @mdogo.
https://github.com/doofinder/pydoof/pull/23

As an alternative I could create another new branch or fix this [here](https://github.com/doofinder/pydoof/tree/BK-723_create-init-request).